docs/app/Views/Usage.js

```diff
@@ -41,22 +42,22 @@ const Usage = () => (
       </p>
       <Header as='h3'>Examples</Header>
       <p>
-          For examples on how to import and use Semantic UI React components,
-          click the code icon next to any example. Here are a few direct links:
+        For examples on how to import and use Semantic UI React components,
+        click the code icon next to any example. Here are a few direct links:
       </p>
       <ul>
-          <li>
-              <a href='http://react.semantic-ui.com/elements/button#button'>Button</a>
-          </li>
-          <li>
-              <a href='http://react.semantic-ui.com/elements/list#list'>List</a>
-          </li>
-          <li>
-              <a href='http://react.semantic-ui.com/views/card#card'>Card</a>
-          </li>
-          <li>
-              <a href='http://react.semantic-ui.com/modules/modal#modal'>Modal</a>
-          </li>
+        <li>
+          <NavLink to='/elements/button#button-example-button'>Button</NavLink>
```

Comment:
These were `<a>` tags which caused a full page reload.  Replaced with `react-router-dom` nav links and fixed paths to point to new anchors.